modules/pubmaticRtdProvider.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+
+- Module Name: PubMatic RTD Provider
+- Module Type: RTD Adapter
+- Maintainer: header-bidding@pubmatic.com
+
+## Description
+
+This RTD module, provided by PubMatic, is used to set dynamic floors within Prebid.
```

Comment:
Please add details about how is it different from the existing floor module in Orebid?